addons/mail/static/src/components/chatter_topbar/chatter_topbar.js

```diff
@@ -72,7 +72,7 @@ export class ChatterTopbar extends Component {
             },
             res_id: false,
         };
-        return this.env.bus.trigger('do-action', {
+        return owl.Component.env.bus.trigger('do-action', {
```

Comment:
Isn't there a way to perform `doAction` in the new env?

Reply:
yes.
```js
// in setup
this.actionService = useService("action")
// elsewhere:
this.actionService.doAction(action)
```

Reply:
Some tests rely on catching the bus event though, I'm ok for the change but it will likely require more adaptations (unless the same event is sent on the same bus with the new syntax?)